client/src/app/s-degree-plan-editor/s-degree-plan-editor.component.html

```diff
@@ -176,33 +178,37 @@
           </div>
           <div class="add-semester-card">
             <button
-              *ngIf="year.isAddingSemester === false"
+              *ngIf="year.newSemesterWidget.active === false"
               mat-button
               (click)="toggleAddingSemester(yearIndex)"
               class="add-semester-font-clr w100"
             >
               + ADD A SEMESTER FOR {{ year.beginning }} - {{ year.ending }}
             </button>
-            <mat-form-field class="mr8" *ngIf="year.isAddingSemester === true">
+            <mat-form-field
+              class="mr8"
+              *ngIf="year.newSemesterWidget.active === true"
+            >
               <mat-label>Semester</mat-label>
-              <mat-select>
-                <mat-option value="SUMMER">Summer</mat-option>
-                <mat-option value="FALL">Fall</mat-option>
-                <mat-option value="SPRING">Spring</mat-option>
-                <mat-option value="WINTER">Winter</mat-option>
+              <mat-select [(ngModel)]="year.newSemesterWidget.termSelect">
```

Comment:
Used ngModel to connect the select menu to the newSemesterWidget property in the plan variable of s-degree-plan-editor component.